spec/system/employee/amenity_groups_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe "amenity_group", type: :system do
+  describe "アメニティーグループの新規登録の画面" do
+    let(:employee) { create(:employee) }
+    before do
+      sign_in employee, scope: :employee
+      visit new_employees_amenity_group_path
+    end
+
+    context "入力フォームに文字列が入っている時" do
+      let(:amenity_group) { build(:amenity_group) }
+      it "新規作成が無事できること" do
+        fill_in "グループネーム", with: amenity_group.name
+        fill_in "内容", with: amenity_group.content
```

Comment:
system specはDRYである必要なく、基本的にユーザーの操作を再現する必要があると思っています！
現状だと、グループネームに何が入力されたのかわからない状態になっているため、万が一`amenity_group.name`がnilでも空文字でもなぜテストが失敗するのか気づきにくくなってしまいます。

```suggestion
        fill_in "グループネーム", with: 'アメニティセットA'
        fill_in "内容", with: 'ドライヤー(Refa）、コーム、コットン'
```

みたいに実際に入力するであろう値を書いた方がより堅牢なテストになると思います

Reply:
レビューありがとうございます！
なるほどそうなんですね‼️
勉強になります🙇‍♀️